admin/kernel-syncdata-service/src/main/java/io/mosip/kernel/syncdata/websub/dto/EventModel.java

```diff
@@ -0,0 +1,12 @@
+package io.mosip.kernel.syncdata.websub.dto;
```

Comment:
same as above can we not have it as part of kernel-core for all websub based events

Reply:
moved models to kernel-core